cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -128,5 +128,13 @@ share
     deriving Eq
     deriving Show
 
+  -- The bit of the pool data on the chain.
+  -- This doesn't leave the internal database.
+  PoolOnData
```

Comment:
Why `PoolOnData` ? Can we come up with a better name or provide a better comment to explain what this does?

Reply:
Pool online data - PoolOnData.
The comment can be updated, not sure what information is missing.

Reply:
Updated.

Reply:
In this case i wonder if `PoolOnlineData` may not be a better name

Reply:
Kevin's schema calls this `TxMetadata`. I suspect `PoolMetadata` might not be the best name.